docs/CONTRIBUTING.md

```diff
@@ -146,6 +146,10 @@ configured by the user.
 
 ### Writing Options<a name="writing-options"></a>
 
+> \[!IMPORTANT\] When writing options for any Nix module, do NOT make any option
```

Comment:
If I'm not mistaken the admoniation (`\[!IMPORTANT\]`) needs to be its own line.

Reply:
Yup, this was a mistake from a previous commit that I made, I don't know what went wrong with the formatting. Also, I don't think using `README.md` and `CONTRIBUTING.md`as pages are a good idea in hindsight, so I just took it out.